packages/block-library/src/navigation/edit/manage-menus-button.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+import { Button } from '@wordpress/components';
+import { memo } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+const ManageMenusButton = ( { className = '', disabled } ) => (
+	<Button
+		variant="link"
+		disabled={ disabled }
+		className={ className }
+		href={ addQueryArgs( 'edit.php', {
+			post_type: 'wp_navigation',
+		} ) }
+	>
+		{ __( 'Manage menus' ) }
+	</Button>
+);
+
+export default memo( ManageMenusButton );
```

Comment:
Rendering this component shouldn't be expensive. We can probably remove `memo` here. What do you think?

Reply:
what is the downside of using memo?

Reply:
Using it won't make the editor slower unless we start wrapping every component. In this case, it probably won't make things faster.

Here's a good quote from new React docs that summarizes this better than me 😅:

> Optimizing with memo  is only valuable when your component re-renders often with the same exact props, and its re-rendering logic is expensive. If there is no perceptible lag when your component re-renders, memo is unnecessary.

See the "Should you add memo everywhere?" section: https://beta.reactjs.org/apis/react/memo.

Reply:
done